<ui version="4.0" >
 <class>ClockBase</class>
 <widget class="QWidget" name="ClockBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Clock</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QAnalogClock" native="1" name="analogClock" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>13</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QFrame" name="Frame3" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="clockLcd" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>1</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="segmentStyle" >
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="clockAmPm" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="Frame4" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="date" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAnalogClock</class>
   <extends>QWidget</extends>
   <header>qanalogclock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
